src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -420,6 +426,20 @@ internal void WorkerInitResponse(GrpcEvent initEvent)
                 _isWorkerApplicationInsightsLoggingEnabled = true;
             }
 
+            // If http proxying is enabled, we need to get the proxying endpoint of this worker
+            var httpUri = _workerCapabilities.GetCapabilityState(RpcWorkerConstants.HttpUri);
```

Comment:
Should the feature flag check be moved around _all_ of the HTTP proxy logic?

Reply:
The other logic is accessed through the _IsHttpProxyingWorker_ var which can only be set to true here when the feature flag is checked - I can do that as well but I was concerned about excessive checking since it would occur in every invocation + invocation response.

Reply:
Discussed w/ Naren we think it's cheap enough and the extra protection is good